src/Xamarin.Android.Build.Tasks/Tasks/AndroidToolTask.cs

```diff
@@ -105,7 +105,7 @@ protected override void LogEventsFromTextOutput (string singleLine, MessageImpor
 \s*
 (?<message>.*)
 $
-", RegexOptions.Compiled | RegexOptions.ExplicitCapture | RegexOptions.IgnorePatternWhitespace);
+", RegexOptions.Compiled | RegexOptions.ExplicitCapture | RegexOptions.IgnorePatternWhitespace, RegexOptions.IgnoreCase);
```

Comment:
@dellis1972 I think this needs `|` instead of `,`

Reply:
http://gph.is/RYv1nB